scripts/preprocess_lipsync.py

```
@@ -0,0 +1,1074 @@
+
+    Returns:
+        MouthPositionData形式の辞書
+    """
+    # モデルをダウンロード
+    script_dir = Path(__file__).parent
+    model_path = download_model(script_dir / "models")
+
+    # 動画を開く
+    cap = cv2.VideoCapture(video_path)
+    if not cap.isOpened():
+        raise ValueError(f"動画を開けません: {video_path}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)